core/src/Kokkos_CopyViews.hpp

```diff
@@ -3478,7 +3478,9 @@ void check_view_ctor_args_create_mirror() {
 // create a mirror
 // private interface that accepts arbitrary view constructor args passed by a
 // view_alloc
-template <class T, class... P, class... ViewCtorArgs>
+template <class T, class... P, class... ViewCtorArgs,
+          class = std::enable_if_t<
+              std::is_void<typename ViewTraits<T, P...>::specialize>::value>>
```

Comment:
```suggestion
              std::is_void_v<typename ViewTraits<T, P...>::specialize>>>
```

Reply:
This part was removed from @masterleinad review. I followed your recommendation, however.